<ui version="4.0" >
 <class>TabPrinter</class>
 <widget class="QWidget" name="TabPrinter" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TabPrinterBase</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item>
    <widget class="Q3GroupBox" name="destinationGroup" >
     <property name="title" >
      <string>Print Destination</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item>
         <widget class="QComboBox" name="defaultPrinter" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>191</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="useAltPrintCommand" >
        <property name="text" >
         <string>Alternative Printer Command</string>
        </property>
        <property name="shortcut" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Command:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="printerCommand" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabOptions" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>10</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="0" column="1" >
        <widget class="Q3ButtonGroup" name="groupPostscript" >
         <property name="title" >
          <string>Postscript Options</string>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>10</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item>
           <widget class="QComboBox" name="psLevel" >
            <item>
             <property name="text" >
              <string>Level 1</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Level 2</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Level 3</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="printColor" >
            <property name="text" >
             <string>Print in Color if Available</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="printGray" >
            <property name="text" >
             <string>Print in Grayscale</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="Q3GroupBox" name="groupPage" >
         <property name="title" >
          <string>Page</string>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>10</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item>
           <widget class="QCheckBox" name="mirrorH" >
            <property name="text" >
             <string>Mirror Page(s) Horizontal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mirrorV" >
            <property name="text" >
             <string>Mirror Page(s) Vertical</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="setMedia" >
            <property name="text" >
             <string>Set Media Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="setClip" >
            <property name="text" >
             <string>Clip to Page Margins</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="Q3GroupBox" name="groupColor" >
         <property name="title" >
          <string>Color</string>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>10</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item>
           <widget class="QCheckBox" name="doGCR" >
            <property name="text" >
             <string>Apply Under Color Removal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="convertSpots" >
            <property name="text" >
             <string>Convert Spot Colors to Process Colors</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="doOverprint" >
            <property name="text" >
             <string>Force Overprint Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useICC" >
            <property name="text" >
             <string>Apply ICC Profiles</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="Q3ButtonGroup" name="groupSeparations" >
         <property name="title" >
          <string>General</string>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>10</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item>
           <widget class="QRadioButton" name="buttonNormal" >
            <property name="text" >
             <string>Print Normal</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="buttonSeparations" >
            <property name="text" >
             <string>Print Separations</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage" >
      <attribute name="title" >
       <string>Marks &amp;&amp; Bleeds</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>10</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item>
        <widget class="Q3GroupBox" name="groupMarks" >
         <property name="title" >
          <string>Printer Marks</string>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>10</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="cropMarks" >
            <property name="text" >
             <string>Crop Marks</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="bleedMarks" >
            <property name="text" >
             <string>Bleed Marks</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="registrationMarks" >
            <property name="text" >
             <string>Registration Marks</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="textLabel2" >
            <property name="text" >
             <string>Offset:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="ScrSpinBox" native="1" name="offsetValue" />
          </item>
          <item row="0" column="1" colspan="2" >
           <widget class="QCheckBox" name="colorMarks" >
            <property name="text" >
             <string>Color Bars</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Q3ButtonGroup" name="groupBleed" >
         <property name="title" >
          <string>Bleed Settings</string>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>10</number>
          </property>
          <property name="spacing" >
           <number>5</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel3" >
            <property name="text" >
             <string>Top:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="textLabel4" >
            <property name="text" >
             <string>Bottom:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="ScrSpinBox" native="1" name="bleedTop" />
          </item>
          <item row="1" column="1" >
           <widget class="ScrSpinBox" native="1" name="bleedBottom" />
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="textLabel5" >
            <property name="text" >
             <string>Left:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="textLabel6" >
            <property name="text" >
             <string>Right:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="ScrSpinBox" native="1" name="bleedLeft" />
          </item>
          <item row="1" column="3" >
           <widget class="ScrSpinBox" native="1" name="bleedRight" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QWidget</extends>
   <header>scrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >mspinbox.h</include>
  <include location="local" >mspinbox.h</include>
  <include location="local" >mspinbox.h</include>
  <include location="local" >mspinbox.h</include>
  <include location="local" >mspinbox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
